contents/data/sql/SQL.md

```diff
@@ -0,0 +1,241 @@
+
+Here are some recommended steps that you can use to start learning SQL:
+
+- Understand what a database and [<tooltip content="DataBase Management System">DBMS</tooltip>]({{baseUrl}}/contents/data/databases/databases.html) are. More specifically, understand the basic concepts of [RDBMS](https://www.tutorialspoint.com/sql/sql-rdbms-concepts.htm). This is not _essential_, but will give you a better high-level understanding before diving into programming.
+
+- Khan Academy offers a very good [SQL course](https://www.khanacademy.org/computing/computer-programming/sql) for free. It has a mix of videos, text, and exercises that you can choose to do at your own pace. It is well organized and even provides summary notes.
+
+- W3Schools offers an interactive written tutorial for [SQL](https://www.w3schools.com/sql/default.asp).
+
+- You can also use a [SQL Quick Reference Guide](https://www.w3schools.com/sql/sql_ref_mysql.asp) to look up information quickly once you're familiar with the language.
+
+- You can also take a look at [40 important SQL queries](https://bytescout.com/blog/20-important-sql-queries.html) to get a quick overview (or refresher) of SQL.
+
+---
+
+## Where to go from here?
+
+Look into SQL distributions and decide which one you want to work with. You can start by comparing the 3 distributions mentioned [here](https://blog.udemy.com/oracle-vs-mysql-vs-sql-server/) (and [here](https://db-engines.com/en/system/Microsoft+SQL+Server%3BMySQL%3BOracle)) and work from there. Then, try to set up a RDBMS and integrate it with an application. This will give you good experience and exposure to how databases are used in practice.
+
+For example:
+
+  - [Guru99](https://www.guru99.com/introduction-to-database-sql.html) has a good DBMS guide that uses MySQL. It covers the entire learning process, from basic database concepts to setting up and using MySQL.
+
+  - Integrate a database (MySQl, PostgreSQL etc.) with your preferred language. For MySQL, you can use the [mysql](http://www.mysqltutorial.org/mysql-nodejs/) module for Node.js, [JDBC](http://www.mysqltutorial.org/mysql-jdbc-tutorial/) for Java, and [MySQL Connector](http://www.mysqltutorial.org/python-mysql/) for Python.
+
+  - W3Schools have interactive written tutorials such as [MySQL Node.js](https://www.w3schools.com/nodejs/nodejs_mysql.asp) and [MySQL Python](https://www.w3schools.com/python/python_mysql_getstarted.asp), for learning how to integrate MySQL with Node.js and Python respectively.
+
+---
+
+## References
+
+- [SQL capabilities](https://www.databasejournal.com/sqletc/article.php/3915331/Top-10-SQL-Hierarchical-Data-Processing-Capabilities.htm)
+- [Advantages and disadvantages of SQL](https://www.quora.com/What-are-the-advantages-and-disadvantages-of-SQL)
+- [MySQL and PostgreSQL differences](http://mwiki.gichd.org/IM/Difference_MySQL_PostGreSQL)
+- [Execution order of SQL query](https://www.designcise.com/web/tutorial/what-is-the-order-of-execution-of-an-sql-query)
```

Comment:
Are these already linked somewhere in the main body? In that case no need to list them again. If they are additional resources, you should give your own take of the each.